website_product_subscription/controllers/subscribe_form.py

```diff
@@ -70,13 +71,18 @@ def normalize_form_data(self):
         else:
             self.qcontext["is_gift"] = False
 
+        if self.qcontext.get("subscriber_address_checked", False) == u"True":
```

Comment:
Are you sure that `subscriber_address_checked` will always contain `"True"` ? Normally, the fact that the entry is present in the `request.params` means that the checkbox is checked, ignoring what is stored in it.

Reply:
Value could be "on" instead of True.

Reply:
Ce n'est pas une checkbox dans le formulaire mais un champ `hidden` que je set moi-même

```
            <input type="hidden"
                   name="subscriber_address_checked"
                   t-att-value="subscriber_address_checked"/>
```